gui/Graph.cpp

```diff
@@ -0,0 +1,25 @@
+#include "Graph.h"
+
+/*
+TODO: the graph class should have a list/vector of all objects to draw and
+2 functions to manage the objects: setObjects (deletes all other objects and
```

Comment:
The way it is now, we would have to cheat with `Point` as they are no "Objects".
You already mentioned that we could add them as very small circles, this would be something we could implement inside of the "add / set Objects"-methods.